components/CreateProfileV2.js

```diff
@@ -25,8 +25,7 @@ const messages = defineMessages({
     defaultMessage: 'Set up your personal details to continue',
   },
   newsletter: {
-    id: 'newsletter.label',
```

Comment:
Because we've changed this string to `Subscribe to our monthly newsletter` it's a new string that doesn't exist in the lang files. So we cannot keep the same `id`; either we have to change the id or remove it. I thought removing it is simpler and consistent with the rest of the message strings.

Reply:
>  So we cannot keep the same id; either we have to change the id or remove it. 

This is not true.

Reply:
What I meant was the if we keep the same id, we will get the following `FormatJs` error;

```
Error: [FormatJS CLI] Duplicate message id: "newsletter.label", but the `description` and/or `defaultMessage` are different.
```
Because `newsletter.label` is used elsewhere.

Reply:
Why should we keep the old one and maintain 2 different strings for _basically_ the same thing?

Reply:
> Why should we keep the old one and maintain 2 different strings for _basically_ the same thing?

Because `newsletter.label` is used in `CreateProfile.js`. After, https://github.com/opencollective/opencollective-frontend/pull/7908 is merged this duplication will go away.

Reply:
@SudharakaP what about `/pages/expense.js` ?

Reply:
Ops, this one I missed somehow. Sorry about that. I've added the same string to `/pages/expense.js`. 🤯